mobile/lib/modules/asset_viewer/ui/exif_bottom_sheet.dart

```diff
@@ -91,6 +98,107 @@ class ExifBottomSheet extends HookConsumerWidget {
       return text.isEmpty ? null : Text(text);
     }
 
+    buildDragHeader() {
+      return Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: const [
+          SizedBox(height: 12),
+          Align(
+            alignment: Alignment.center,
+            child: CustomDraggingHandle(),
+          ),
+          SizedBox(height: 12),
+        ],
+      );
+    }
+
+    buildLocation() {
+      // Guard no lat/lng
+      if (!showMap) {
+        return Container();

```

Comment:
We should use `SizedBox` instead https://dart-lang.github.io/linter/lints/sized_box_for_whitespace.html

Reply:
It's not whitespace; it's just an empty container. It won't be rendered as it has 0 width and 0 height. I don't get any linter errors on my end, but I'm happy to change it to a `SidedBox` if you'd like!